ui/app/src/components/DashboardList/DashboardList.tsx

```diff
@@ -83,21 +83,38 @@ export function DashboardList(props: DashboardListProperties) {
   );
 
   const handleDashboardDuplication = useCallback(
-    (dashboardSelector: DashboardSelector) => {
+    (dashboardInfo: DashboardInfo) => {
```

Comment:
maybe we could have something like that instead: 

```typescript
(info: DashboardInfo | EphemeralDashboardInfo) => {}
```

That would avoid to modify the datamodel of the dashboard which is not that great I think.